hack/update-generated-docs.sh

```diff
@@ -25,6 +25,9 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+# This only runs on master
+[[ $(git rev-parse --abbrev-ref HEAD) == "master" ]] || exit
```

Comment:
Right, thanks for pointing that out.  I should rather check for a positive on 'release-' vs. !master.
So the rewrite you did on this makes these generated docs overwrite with the placeholder text wherever they are run.
I added you as a reviewer because based on the work you did there may be a better approach. 
Ultimately, we want to be able to run 'update-all.sh' anywhere and have it do the right thing.  
Currently, it undoes the work of versionize-docs.sh.